composeApp/src/commonMain/kotlin/org/dots/game/views/ControlButtons.kt

```diff
@@ -0,0 +1,207 @@
+package org.dots.game.views
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Checkbox
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import dotsgame.composeapp.generated.resources.*
+import org.dots.game.Platform
+import org.dots.game.Tooltip
+import org.dots.game.UiSettings
+import org.dots.game.core.*
+import org.dots.game.localization.Strings
+import org.jetbrains.compose.resources.DrawableResource
+import org.jetbrains.compose.resources.painterResource
+
+@Composable
+fun ControlButton(
+    text: String,
+    icon: DrawableResource,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true
+) {
+    Tooltip(text) {
+        Button(
+            onClick = onClick,
+            modifier = modifier,
+            enabled = enabled
+        ) {
+            Icon(
+                painterResource(icon),
+                contentDescription = text,
+                modifier = Modifier.size(20.dp)
+            )
+        }
+    }
+}
+
+@Composable
+fun GameControlButtons(
+    strings: Strings,
+    modifier: Modifier = Modifier,
+    buttonModifier: Modifier = Modifier,
+    onNewGame: () -> Unit,
+    onReset: () -> Unit,
+    onLoad: () -> Unit,
+    onSave: () -> Unit,
+    onSettings: () -> Unit,
+    onAiSettings: () -> Unit,
+    showAiSettings: Boolean
+) {
+    Row(modifier) {
+        ControlButton(strings.new, Res.drawable.ic_new_game, onNewGame, buttonModifier)
+        ControlButton(strings.reset, Res.drawable.ic_refresh, onReset, buttonModifier)
+        ControlButton(strings.load, Res.drawable.ic_folder_open, onLoad, buttonModifier)
+        ControlButton(strings.save, Res.drawable.ic_save, onSave, buttonModifier)
+        ControlButton(strings.settings, Res.drawable.ic_settings, onSettings, buttonModifier)
+        if (showAiSettings) {
+            ControlButton(strings.aiSettings, Res.drawable.ic_smart_toy, onAiSettings, buttonModifier)
+        }
+    }
+}
```

Comment:
The idea looks reasonable, however, I've decided to postpone it because I'm not sure how the buttons should be grouped. Ideally, we should introduce "Game" and "Editing" modes. The "Game" mode should have minimal UI. "Editing" mode should have more controls.